pkg/printers/humanreadable.go

```diff
@@ -535,7 +535,10 @@ func (h *HumanReadablePrinter) PrintTable(obj runtime.Object, options PrintOptio
 // different from lastType) including all the rows in the object. It returns the current type
 // or an error, if any.
 func printRowsForHandlerEntry(output io.Writer, handler *handlerEntry, obj runtime.Object, options PrintOptions, includeHeaders bool) error {
-	if includeHeaders {
+	args := []reflect.Value{reflect.ValueOf(obj), reflect.ValueOf(options)}
```

Comment:
This will cause a panic if `handler.printRows == false`. When this field is true, an extra arg for `output` is added to the list of args. See how this is currently handled below

Reply:
thanks :)